system/Commands/Database/ShowTableInfo.php

```diff
@@ -0,0 +1,236 @@
+<?php
+
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+namespace CodeIgniter\Commands\Database;
+
+use CodeIgniter\CLI\BaseCommand;
+use CodeIgniter\CLI\CLI;
+use Config\Database;
+
+/**
+ * Get table data if it exists in the database.
+ */
+class ShowTableInfo extends BaseCommand
+{
+    /**
+     * The group the command is lumped under
+     * when listing commands.
+     *
+     * @var string
+     */
+    protected $group = 'Database';
+
+    /**
+     * The Command's name
+     *
+     * @var string
+     */
+    protected $name = 'db:table';
+
+    /**
+     * the Command's short description
+     *
+     * @var string
+     */
+    protected $description = 'Retrieves information on the selected table.';
+
+    /**
+     * the Command's usage
+     *
+     * @var string
+     */
+    protected $usage = 'db:table <table_name> [options]';
+
+    /**
+     * The Command's arguments
+     *
+     * @var array<string, string>
+     */
+    protected $arguments = [
+        'table_name' => 'The table name for show info',
+    ];
+
+    /**
+     * The Command's options
+     *
+     * @var array<string, string>
+     */
+    protected $options = [
+        '--show'               => 'List the names of all database tables.',
+        '--metadata'           => 'Retrieves list of containing field information.',
+        '--desc'               => 'Sort direction DESC of the tables Rows.',
+        '--limit-rows'         => 'Limit the number of rows. [Default = 10]',
+        '--limit-fields-value' => 'Custom length Limit of field values. [Default = 15]',
```

Comment:
At first reading, I don't understand what this option does.

Reply:
I think `--limit-columns` is better. 
`Limit the number of columns. [Default = 15]` ?